src/users/users.controller.ts

```diff
@@ -1,13 +1,31 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { User } from '@prisma/client';
+import {
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import { UsersEntity } from './entities/users.entity';
 
+@ApiTags('users')
 @Controller('users')
 export class UsersController {
   constructor(private usersService: UsersService) {}
 
   @Get()
+  @ApiOperation({ summary: 'Get users list' })
+  @ApiOkResponse({ type: UsersEntity, isArray: true })
   getUsers(): Promise<User[]> {
     return this.usersService.getUsers();
   }
+
+  @Get(':id')
+  @ApiOperation({ summary: 'Get users by id' })
+  @ApiOkResponse({ type: UsersEntity })
+  @ApiNotFoundResponse({ description: 'User not found' })
+  getUser(@Param('id') id: string): Promise<User> {
```

Comment:
I'd avoid returning `User` model from Prisma and instead create DTO classes for requests / responses. This way we are separating database model from what we want to return in a presentation layer. Here you could return `Promise<GetUserResponse>` and `GetUserResponse` would be a class defined in `/users/responses/get-user.response.ts` or `/users/dto/get-user-response.dto.ts`.